model/RepositoryService.php

```diff
@@ -180,21 +180,9 @@ public function restore(Revision $revision)
             $this->getModel()->getRdfInterface()->add($triple);
         }
 
-        $item = $this->getQtiService()->getDataItemByRdfItem($resource);
-        $this->getUpdatedItemEventDispatcher()->dispatch($item, $resource);
```

Comment:
If I remember well, this event is here to create related MediaRelation. I agree to remove it but we need to find an alternative to populate these relations

Reply:
yes, that will come in a later PR, 
for the moment this is a blocker issue

Reply:
Was this removed because is slow? What is the relation with current bug?

Reply:
this was breaking the flow, because in case of tests or assets we were expecting ``$item`
and that can't be true in those cases

Reply:
I got it now. Would not be the case to check if `$resource` is an Item then?

Reply:
@bartlomiejmarszal will solve this part later. So I am resolving.